tests/phpunit/CRM/Core/BAO/FinancialTrxnTest.php

```diff
@@ -126,8 +126,6 @@ public function testGetExPartialPaymentTrxn() {
       civicrm_api3('Contribution', 'completetransaction', array('id' => $contribution['id']));
     }
 
-    $this->assertEquals('100.00', $trxn->total_amount, 'Amount does not match.');
```

Comment:
This assertion is gone because...?

Reply:
It wasn't really testing what was created & it was a pain to change (data was still the same)